crates/hyperswitch_connectors/src/connectors/braintree/transformers.rs

```diff
@@ -2038,7 +2038,7 @@ pub struct DisputeEvidence {
 pub(crate) fn get_dispute_stage(code: &str) -> Result<enums::DisputeStage, errors::ConnectorError> {
     match code {
         "CHARGEBACK" => Ok(enums::DisputeStage::Dispute),
-        "PRE_ARBITATION" => Ok(enums::DisputeStage::PreArbitration),
+        "PRE_ARBITRATION" => Ok(enums::DisputeStage::PreArbitration),
```

Comment:
This would need to be checked with connector docs.

Reply:
i checked the brain tree docs and the changes are indeed correct.

<img width="580" alt="image" src="https://github.com/user-attachments/assets/057db41d-94fd-4c69-bd83-0d3050a8ced4" />

link: https://developer.paypal.com/docs/api/customer-disputes/v1/#disputes_provide-supporting-info